### PR TITLE
feat: add mdx and about page

### LINKED ISSUE #33: Description of project / Learn more
The `Learn more` button on the homepage should link to another page `/about` to explain what the project is to new visitors to the site.

### PR DESCRIPTION
**Pull request by exu3**
closes #33

### CODE REVIEW

**Comment by exu3** [comment]
z-index: it no work :sadge:

@sarthaktexas should i get the text to overlap with the svg or move it down?

**Comment by zaida04** [comment]
I think the design would look really cool if the text would overlap with the svg. Otherwise it just feels like a lot of blank space.

File: styles/globals.css [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
 @tailwind components;
 @tailwind utilities;
 
 .w-fitcontent {
     width: fit-content;
-}
\ No newline at end of file
+}
+
+@layer base {
```
Comment (sarthaktexas) [inline styles/globals.css]: nonono. i didn't mention this cause i thought you knew. remove this and do something like this in `_app.js`:

```js
import '../styles/globals.css';
import { MDXProvider } from '@mdx-js/react'
import MDXComponents from '../components/MDXComponents'
import { AuthProvider } from '../lib/auth'

function MyApp({ Component, pageProps }) {
  return (
    <AuthProvider>
      <MDXProvider components={MDXComponents}>
          <Component {...pageProps} />
      </MDXProvider>
    </AuthProvider>
  );
}

export default MyApp;
```

and then in `MDXComponents.js`:
```js

const CustomLink = (props) => {
    const href = props.href;
    const isInternalLink = href && (href.startsWith('/') || href.startsWith('#'));

    if (isInternalLink) {
        return (
            <Link href={href}>
                <a {...props} />
            </Link>
        );
    }

    return <a target="_blank" rel="noopener noreferrer" {...props} />;
};

const CustomHeader = (props) => {
    return (
        <h1 className="text-xl font-bold mt-4 mb-1" {...props} />
    )
}

const MDXComponents = {
    h1: CustomHeader,
    a: CustomLink,
};

export default MDXComponents;
```
> By the way, the link component above automatically put `next/link` when it's a client side redirect :)

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
Other than that comment I think it's fine. And yes, overlap them, I want to see how it looks.

**Comment by awlevin** [comment]
#### 📝Changed routes:


| [**/about**](https://open-lrw81n9ae-thinkfiveable.vercel.app/about) | [**/**](https://open-lrw81n9ae-thinkfiveable.vercel.app/) |
|:-:|:-:|
|<a href="https://open-lrw81n9ae-thinkfiveable.vercel.app/about">Screenshot of /about</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/AF7VfkmG-CXVgs1pPJ36hMLASf4w6acNAAu1vDad8x-J-qd3wIAjy6fEw-IEefWOQB-4yS7OBDN_V3bnkixStw.png">(view full size)</a> |<a href="https://open-lrw81n9ae-thinkfiveable.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/LTEJ4v71mrRXvxATr01D_U-JApQVoxHnXPKHK6ZSNbNFDTto3jsZ5ouf8Cg4uttCZ3aJBo9dBRNSSdWC5GoWLw.png">(view full size)</a> |







Commit 6bf7b0a24e139f493324f856f2da6a31b0bc0bd9 (https://open-lrw81n9ae-thinkfiveable.vercel.app).

**Comment by zaida04** [comment]
I'd like us to have a heading on top of https://github.com/ThinkFiveable/open/pull/36/files#diff-62560091290dc9edebbff8bb0ff6f88a3eacb7ef5fbec9e4b320ddd54b44d55cR5, otherwise the body text looks out of place.

Can we also make the mouse pointer change when people hover over `Learn More`? Right now the mouse pointer stays the same when hovering over the Learn More button.

**Review by zaida04 — DISMISSED** [review]
Thank you so much Ella! Looks dope 🔥

File: pages/index.js [hunk 2]
```diff
@@ -31,10 +32,12 @@ export default function Home() {
                     <h2 className="font-rubik font-medium text-3xl md:w-2/3 lg:w-2/5">
                         We&#39;re building a community for discovery and learning.
                     </h2>
-                    <div className="font-rubik font-medium bg-black text-center rounded-full px-5 py-1 text-lg text-white w-fitcontent flex flex-row">
-                        <div>Learn more.</div>
-                        <BsArrowRightShort size={30} />
-                    </div>
+                    <Link href="/about">
+                        <div className="font-rubik font-medium bg-black text-center rounded-full px-5 py-1 text-lg text-white w-fitcontent flex flex-row cursor-pointer">
```
Comment (sarthaktexas) [inline pages/index.js]: Make this an anchor tag

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

**Review by sarthaktexas — APPROVED** [review]
'

**Review by zaida04 — APPROVED** [review]

**Pull request merged**